docs/getting-started-guides/docker.md

```diff
@@ -56,6 +54,8 @@ docker run \
 
 > If you would like to mount an external device as a volume, add `--volume=/dev:/dev` to the command above. It may however, cause some problems described in [#18230](https://github.com/kubernetes/kubernetes/issues/18230)
 
+> Other architectures than `amd64` are experimental and sometimes unstable, but feel free to try them out! Valid values: `arm`, `arm64` and `ppc64le`. ARM is available with Kubernetes version `v1.3.0-alpha.2` and higher. ARM 64-bit and PowerPC 64 little-endian are available with `v1.3.0-alpha.3` and higher. Track as progress is being made in [this proposal](https://github.com/kubernetes/kubernetes/issues/17981)
```

Comment:
Hahaha :smile:
Thanks for the comments, it's late at night here already so it's good to have someone thinking about how to say things the best way :)